hatch_vcs/version_source.py

```diff
@@ -46,14 +46,13 @@ def config_raw_options(self):
             self.__config_raw_options = raw_options
 
         return self.__config_raw_options
-
-    def get_version_data(self):
+    
+    @property
+    def setuptools_scm_config(self):
         from copy import deepcopy
 
-        from setuptools_scm import get_version
-
         config = deepcopy(self.config_raw_options)
-        config['root'] = self.root
```

Comment:
I believe this fix is incomplete as root for hatch and setuptools_scm are distinct ideas